commands/system/help.js

```diff
@@ -0,0 +1,211 @@
+    if (!type) {
+      const description = `Command category list\n\nUse \`${message.settings.prefix}help <category>\` to find commands for a specific category`;
+      const output = sorted.filter(c => !(level < 10 && c.help.category === 'Owner') || !(c.help.category === 'NSFW' && !message.channel.nsfw)).map((c) => {
+        const cat = c.help.category;
+        if (currentCategory !== cat && !type) {
+          currentCategory = cat;
+          return `\n\`${message.settings.prefix}help ${cat.toLowerCase()}\``;
+        }
+      }).join('');
+      embed.setDescription(description)
+        .addField('Categories', output);
+    } else {
+      sorted.forEach((c) => {
+        if (c.help.category.toLowerCase() === type.toLowerCase()) {
+          n++;
+        }
+      });
+      let output = '';
+      const pg = Number(page) && Number(page) <= Math.ceil(n / perpage) ? Number(page) : 1;
+      for (const c of sorted.values()) {
+        if (c.help.category.toLowerCase() === type.toLowerCase()) {
+          if (c.help.category === 'NSFW' && !message.channel.nsfw) return;
+          if (num < perpage * pg && num > perpage * pg - (perpage + 1)) {
+            if (level < this.client.levelCache[c.conf.permLevel]) return;
+            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
+          }
+          num++;
+        }
+      }
+
+      if (num) {
+        embed.setTitle(`Page ${page}/${Math.ceil(num / perpage)}`)
+          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+          .addField('Commands', output);
+      }
+    }
+
+    if (this.client.commands.has(type) || this.client.aliases.has(type)) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get((type)));
+      if (level < this.client.levelCache[cm.conf.permLevel]) return;
+      embed.setTitle(cm.help.name)
+        .addField('Command description', cm.help.description)
+        .addField('Command usage', `\`${cm.help.usage}\``)
+        .addField('Command aliases', cm.conf.aliases.length === 0 ? 'None' : cm.conf.aliases.join(', '));
+    }
+
+    const msg2 = await message.channel.send(embed);
+    const totalpages = Math.ceil(num / perpage);
+    if (!message.guild.me.hasPermission(['MANAGE_MESSAGES'])) {
+      await message.channel.send('I don"t have permission to remove reactions, please do this manually.');
+    }
+
+    if(msg2.embeds[0].title) {
+      if (msg2.embeds[0].title.startsWith('Page')) {
+        for (const emoji of emojis) await msg2.react(emoji);
+      }
+    }
+
+    const nextpage = msg2.createReactionCollector(
```

Comment:
You're still using a lot of reaction collectors.

Reply:
That’s solved. Need to push a new update when I get home